tools/consensus_from_alignments/consalign.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="aligned_to_consensus" name="Consensus sequence from aligned FASTA" version="@VERSION@">
+    <description></description>
+    <macros>
+        <import>consalign_macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="0.1.4">r-bioseq</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/consensus_from_alignments.R'
+         '$input'
+         '$seqtype'
+         '$method'
+         '$gaps'
+         '$outog'
+         '$output'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input" type="data" format="fasta" label="Input fasta file with at least two sequences"/>
+        <param name="seqtype" type="select" label="Type of sequence" multiple="false" optional="false" display="radio">
+            <option value="DNA" selected="true">DNA</option>
+            <option value="RNA">RNA</option>
+            <option value="AA">Amino Acid</option>
+        </param>
+        <param name="method" type="select" label="Consensus method when differences are found between sequences" multiple="false" optional="false" display="radio">
+            <option value="chr_ambiguity" selected="true">Use an ambiguous nucleotide</option>
+            <option value="chr_majority">Use the most frequent nucleotide</option>
+        </param>
+        <param name="gaps" type="boolean" checked="no" label="Take account of gaps?" help="/!\ If gaps are at the same place in all the sequences don't check 'no', you'll face an error"/>
```

Comment:
If we know the error, we can catch it with the `stdio` tag and include a more descriptive help text in the error message.